### ISSUE TITLE
Flexdatalist not accessible

### ISSUE THREAD

**Issue by rahulbhatt007** [sergiodlopes/jquery-flexdatalist#218]
Hi,

We are using the component as a typeahead component. The component is not accessible. I have tried using Voiceover/NVDA and it doesn't read anything on moving up/down through the results. Nor does it reads the text being written on input field.

I think it is almost completely non-accessible for disabled users. Do you think we can add accessible controls as part of the component?
